### ISSUE TITLE
Unhandled Runtime Error EvalError: Possible side-effect in debug-evaluate with Chromium 102 

### ISSUE THREAD

**Issue by zerochae** [vercel/next.js#37675]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

```
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000
    Binaries:
      Node: 16.14.2
      npm: 8.5.0
      Yarn: 3.2.0
    Relevant packages:
      next: 12.1.5 (12.1.7-canary.38 also)
      react: 18.1.0
      react-dom: 18.1.0
```

### What browser are you using? (if relevant)

```
Chrome  102.0.5005.108
```

### Describe the Bug

1. run project
2. open dev tools
3. type something in console

<img width="957" alt="스크린샷 2022-06-14 13 57 45" src="https://user-images.githubusercontent.com/84373490/173496511-5f9f8e04-4798-4981-af25-cba31424a709.png">

### Expected Behavior

After closing the warning window, you can re-enter the code in the console and run the code.

### To Reproduce

1. refresh(reload)
2. open devtool and type something

**Comment by zerochae** [comment]
https://stackoverflow.com/questions/72396527/evalerror-possible-side-effect-in-debug-evaluate-in-google-chrome


The same is posted on Stack Overflow.

**Comment by balazsorban44** [comment]
I tracked this to a potential Chrome bug: https://github.com/electron/electron/issues/34350#issuecomment-1137720001

If I understand correctly, the fix should be included in `105.0.5117.0` so let's wait and see.

**Comment by cseitz** [comment]
For those who encounter this like I just did; updating chrome fixed it.

After updating to Chrome `Version 103.0.5060.53 (Official Build) (64-bit)` this issue was resolved.
